docs/concepts/why_splinter.md

```diff
@@ -0,0 +1,285 @@
+notifications for pending circuit proposals and lets an application register for
+specific proposal-related events. The Biome component provides user and private
+key management, which helps bootstrap applications that need to add smart
+contract capabilities to an existing infrastructure that doesn't already include
+the concept of transactions signed by a user.
+
+Splinter's experimental features allow optional access to new functionality.
+Because Splinter is undergoing rapid development, new capabilities are initially
+added as experimental Rust features that are exposed by the `splinter` library.
+This lets developers access bleeding-edge features in a controlled way, because
+these features often change before they are stabilized. (Biome is an example of
+a Splinter component that was added as an experimental feature.) Developer
+feedback is strongly encouraged for experimental features.
+
+Splinter provides
+[Gameroom](https://github.com/Cargill/splinter/tree/master/examples/gameroom)
+as a complete example for building smart-contract-based applications with the
+Splinter platform.
+
+## <a name="bookmark-event-subscription"></a> Event Subscription for State Changes
+
+As described earlier, the Scabbard service includes Hyperledger Transact for
+transaction processing and state management. State agreement is achieved with
+the Merkle-Radix tree, allowing multiple nodes or services to prove they have
+the same data down to the last bit, cryptographically.
+
+The Scabbard service implements a [Sawtooth-style event
+system](https://sawtooth.hyperledger.org/docs/core/releases/1.0/architecture/events_and_transactions_receipts.html)
+so that applications can subscribe to events of interest. **State delta export**
+allows an application to materialize the state data from the Merkle-radix tree
+to another database, event stream, log, or other mechanism. This allows
+applications to query and analyze the smart contract state in their own
+business-specific databases.
+
+## <a name="bookmark-complex-data-flow"></a> Complex Patterns for Data Flow
+
+Splinter has clear benefits when compared to traditional EDI systems and ETL
+processes, which provide data- or outcome-focused data flows. Splinter enables
+complex data flows based on transactions as well as data changes, which allows
+partners to collaborate and update the data in a secure and verifiable way.
+
+With Splinter, your application can import data, allow partners to collaborate
+on changes (with verified state agreement), and export changes in the required
+format. Smart contracts allow even more elaborate patterns for data generation
+and updates.
+
+## <a name="bookmark-open-source"></a> Auditable Open Source
+
+Splinter is open source software, licensed under the [Apache License Version
+2.0](https://github.com/Cargill/splinter/blob/master/LICENSE) software license.
+The Splinter source code is freely available and auditable.
+
+
```

Comment:
Some sections end with two blank lines, others with just one; this should be consistent throughout the doc.

Reply:
Deleted extra blank lines.